corehq/blobs/tasks.py

```diff
@@ -8,7 +8,7 @@
 from corehq.blobs import get_blob_db
 
 
-@periodic_task(run_every=crontab(minute=0, hour='0,12'))
+@periodic_task(run_every=crontab(minute='*/5'))
```

Comment:
Curious why this was changed? I think it was deliberately set to be infrequent to keep load to a minimum. The guarantees on blob expiry are loose enough that this should not have been a problem, and nothing should depend on a blob actually being deleted on a strict schedule due to expiry.

Reply:
This adds caching to mobile UCRs for ICDS. I used the blobs because they seemed like the obvious way to cache XML.

> nothing should depend on a blob actually being deleted on a strict schedule due to expiry.

Is this a technical limitation with riak, or is it a limitiation that we have set up blob expirations? I could use set up some other model, but it seemed better to reuse this table.

Reply:
> I used the blobs because they seemed like the obvious way to cache XML.

That's a fine place to cache XML, but presence of a key in riak should not be used to tell if the cached value is valid or not.

There are some technicalities related to which version of a blob may be returned after its value has been changed or deleted. This is due to the eventual consistency design of riak. It's a performance trade-off.

The [`AbstractBlobDB.put` `timeout`](https://github.com/dimagi/commcare-hq/blob/0196adebab4c610e338e60055b0ba77d579fe27d/corehq/blobs/interface.py#L31-L33) parameter is documented as: "There are no guarantees for the maximum time it will live in blob storage."

Reply:
Thanks for the background. Sounds like I'll need to setup a new model or caching via redis or something to get caching working properly.

I've reverted those changes to this PR, so that this can move forward for enikshay. Can always follow up with caching later